packages/stream_chat_localizations/lib/src/stream_chat_localizations_fr.dart

```diff
@@ -0,0 +1,360 @@
+  @override
+  String get addAFileLabel => 'Ajouter un fichier';
+
+  @override
+  String get photoFromCameraLabel => "Photo de l'appareil photo";
+
+  @override
+  String get uploadAFileLabel => 'Transférer un fichier';
+
+  @override
+  String get uploadAPhotoLabel => 'Transférer une photo';
+
+  @override
+  String get uploadAVideoLabel => 'Transférer une vidéo';
+
+  @override
+  String get videoFromCameraLabel => 'Vidéo depuis la camera';
+
+  @override
+  String get okLabel => 'OK';
+
+  @override
+  String get somethingWentWrongLabel => 'Quelque chose a mal tourné';
+
+  @override
+  String get addMoreFilesLabel => "Ajouter d'autres fichiers";
+
+  @override
+  String get enablePhotoAndVideoAccessMessage =>
+      "Veuillez autoriser l'accès à vos photos"
+      '\net vidéos afin de pouvoir les partager avec vos amis.';
+
+  @override
+  String get allowGalleryAccessMessage => "Autoriser l'accès à votre galerie";
+
+  @override
+  String get flagMessageLabel => 'Signaler un message';
+
+  @override
+  String get flagMessageQuestion =>
+      'Voulez-vous envoyer une copie de ce message à une'
+      '\modérateur pour une enquête plus approfondie ?';
+
+  @override
+  String get flagLabel => 'SIGNALER';
+
+  @override
+  String get cancelLabel => 'ANNULER';
+
+  @override
+  String get flagMessageSuccessfulLabel => 'Message signalé';
+
+  @override
+  String get flagMessageSuccessfulText =>
+      'Ce message a été signalé à un modérateur.';
+
+  @override
+  String get deleteLabel => 'SUPPRIMER';
+
+  @override
+  String get deleteMessageLabel => 'Supprimer le message';
+
+  @override
+  String get deleteMessageQuestion =>
+      'Êtes-vous sûr de vouloir supprimer définitivement ce\nmessage ?';
+
+  @override
+  String get operationCouldNotBeCompletedText =>
+      "L'opération n'a pas pu être terminée.";
+
+  @override
+  String get replyLabel => 'Répondre';
+
+  @override
+  String togglePinUnpinText({required bool pinned}) {
+    if (pinned) return 'Détacher de la conversation';
+    return 'Attacher à la conversation';
+  }
+
+  @override
+  String toggleDeleteRetryDeleteMessageText({required bool isDeleteFailed}) {
+    if (isDeleteFailed) return 'Retenter de supprimer le message';
+    return 'Supprimer le message';
+  }
+
+  @override
+  String get copyMessageLabel => 'Copier le message';
+
+  @override
+  String get editMessageLabel => 'Modifier le message';
+
+  @override
+  String toggleResendOrResendEditedMessage({required bool isUpdateFailed}) {
+    if (isUpdateFailed) return 'Renvoyer le message modifié';
+    return 'Renvoyer';
+  }
+
+  @override
+  String get photosLabel => 'Photos';
+
+  String _getDay(DateTime dateTime) {
+    final now = Jiffy(DateTime.now());
+    final date = Jiffy(dateTime);
+
+    if (date.isSame(now, Units.DAY)) {
+      return "aujourd'hui";
+    } else if (now.diff(date, Units.HOUR) < 24) {
+      return 'hier';
+    } else {
+      return 'le ${date.MMMd}'; //hmm lookup french format d/MM/yyyy
```

Comment:
@sachaarbonel Jiffy will do it automatically

Reply:
ohh nice